test/unit/query/builder.js

```diff
@@ -25,97 +35,105 @@ var clientsWithNullAsDefault = {
   oracle: new Oracle_Client(useNullAsDefaultConfig),
   oracledb: new Oracledb_Client(useNullAsDefaultConfig),
   sqlite3: new SQLite3_Client(useNullAsDefaultConfig),
-  mssql: new MSSQL_Client(useNullAsDefaultConfig),
-}
+  mssql: new MSSQL_Client(useNullAsDefaultConfig)
+};
 
 function qb_sqlite() {
-  return clients.sqlite3.queryBuilder()
+  return clients.sqlite3.queryBuilder();
 }
 function raw_sqlite(sql, bindings) {
-  return clients.sqlite3.raw(sql, bindings)
+  return clients.sqlite3.raw(sql, bindings);
 }
 
-function callQueryBuilderCallback(selectedClients, clientType, callback){
-    var client = selectedClients[clientType]
-	
-	return callback(queryBuilderFactory, raw)
-	
-	function queryBuilderFactory(){
-	  return client.queryBuilder()
-	}
-	
-	function raw(sql, bindings) {
-	  return client.raw(sql, bindings)
-	}  
+function callQueryBuilderCallback(selectedClients, clientType, callback) {
+  var client = selectedClients[clientType];
+
+  return callback(queryBuilderFactory, raw);
+
+  function queryBuilderFactory() {
+    return client.queryBuilder();
+  }
+
+  function raw(sql, bindings) {
+    return client.raw(sql, bindings);
+  }
 }
 
 function verifySqlResult(dialect, expectedObj, sqlObj) {
-  Object.keys(expectedObj).forEach(function (key) {
+  (0, _keys2.default)(expectedObj).forEach(function (key) {
```

Comment:
This shouldn't have been changed and none of the similar changes in the rest of the file either.